src/main/scala/org/constellation/API.scala

```diff
@@ -456,10 +456,16 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
           }
         } ~
         path("ready") {
-          def changeStateToReady: IO[Unit] = dao.cluster.setNodeState(NodeState.Ready)
-          def broadcastState: IO[Unit] = dao.cluster.broadcastNodeState()
+          def changeStateToReady: IO[SetStateResult] = dao.cluster.compareAndSet(NodeState.initial, NodeState.Ready)
```

Comment:
Does it mean that I can use this endpoint only if the node is in the initial state?